includes/templates/template_default/templates/tpl_modules_shipping_estimator.php

```diff
@@ -85,11 +85,11 @@
     }else{
 ?>
 <table id="seQuoteResults">
-<?php if ($_SESSION['customer_id'] < 1 ){ ?>
+<?php if (!isset($_SESSION['customer_id']) || $_SESSION['customer_id'] < 1 ){ ?>

```

Comment:
`empty`?

Reply:
```suggestion
<?php if (empty($_SESSION['customer_id'])){ ?>
```
Certainly makes sense to me. I couldn't think of a specific case where the customer_id would be non-integer and less than 0 considering this had specifically evaluated to less than 1....  Don't know impact on anything else that has "used" this code.